docs/source/main_classes/pipelines.rst

```diff
@@ -21,61 +31,75 @@ other pipeline but requires an additional argument which is the `task`.
 
 
 The task specific pipelines
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Parent class: Pipeline
-=========================================
-
-.. autoclass:: transformers.Pipeline
-    :members: predict, transform, save_pretrained
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-TokenClassificationPipeline
+ConversationalPipeline
```

Comment:
Just a reorderin by alphabetical order and put the main Pipeline class at the end since it's more abstract.